### ISSUE TITLE
Pretrained LDG got extreme low accuracy on RAF-DB

### ISSUE THREAD

**Issue by youcaiSUN** [zengqunzhao/EfficientFace#4]
Hi, zengqun

Thanks for your nice work. I ran the code with the provided models  on RAF-DB (I did not modify anything except the data path), however, I got every low accuracy  as below:

> Current learning rate: 0.00010000000000000003
Epoch: [99][ 0/96]	Loss 0.0743 (0.0743)	Accuracy  3.906 ( 3.906)
Epoch: [99][10/96]	Loss 0.0642 (0.0640)	Accuracy  0.000 ( 1.420)
Epoch: [99][20/96]	Loss 0.0661 (0.0641)	Accuracy  0.781 ( 1.749)
Epoch: [99][30/96]	Loss 0.0552 (0.0627)	Accuracy  0.781 ( 1.815)
Epoch: [99][40/96]	Loss 0.0636 (0.0624)	Accuracy  3.906 ( 1.925)
Epoch: [99][50/96]	Loss 0.0739 (0.0627)	Accuracy  1.562 ( 1.915)
Epoch: [99][60/96]	Loss 0.0604 (0.0622)	Accuracy  0.781 ( 1.857)
Epoch: [99][70/96]	Loss 0.0473 (0.0615)	Accuracy  1.562 ( 1.816)
Epoch: [99][80/96]	Loss 0.0638 (0.0612)	Accuracy  3.125 ( 1.823)
Epoch: [99][90/96]	Loss 0.0632 (0.0610)	Accuracy  2.344 ( 1.829)
Test: [ 0/24]	Loss 7.5441 (7.5441)	Accuracy  6.250 ( 6.250)
Test: [10/24]	Loss 16.6608 (14.6800)	Accuracy  0.781 ( 2.841)
Test: [20/24]	Loss 8.7611 (12.6607)	Accuracy  0.000 ( 2.009)
  \* Accuracy 1.825
Current best accuracy: 4.269882678985596
27.060322523117065

I believe there might have bugs for the pretrained LDG. Thus, I directly validate its performance on RAF-DB, I found that  pretrained LDG got extreme low accuracy. The modified validation code and the experiment log are as below:

```
    if args.evaluate:
        # validate(val_loader, model_cla, criterion_val, args)
        validate(val_loader, model_dis, criterion_val, args)
        return
```

> Training time: 06-21 15:24
Test: [ 0/24]	Loss 9.1622 (9.1622)	Accuracy  5.469 ( 5.469)
Test: [10/24]	Loss 20.1950 (17.4116)	Accuracy  2.344 ( 3.409)
Test: [20/24]	Loss 9.6880 (14.8375)	Accuracy  0.781 ( 2.604)
 *** Accuracy 2.347  *** 
 
Could you give me help? Thanks very much!

**Comment by youcaiSUN** [comment]
I found the solution to my question. I did not restructure the dataset like yours. After restructuring, everything goes right. Thanks for your sharing again!

**Comment by zhang6666** [comment]
How do you  restructure the dataset?

**Comment by EHuba94** [comment]
> I found the solution to my question. I did not restructure the dataset like yours. After restructuring, everything goes right. Thanks for your sharing again!

What were your actions?
